src/models/sessionBus.js

```diff
@@ -180,7 +183,9 @@ var FlatpakSessionBusModel = GObject.registerClass({
     saveToKeyFile(keyFile) {
         const group = this.constructor.getGroup();
         Object.entries(this._overrides).forEach(([key, value]) => {
-            keyFile.set_value(group, key, value);
+            if (this._expression.test(key)) {
```

Comment:
For consistency, instead of doing this here, can you move this to `updateFromProxyProperty`? Like I do it here https://github.com/tchx84/Flatseal/blob/master/src/models/variables.js#L86